src/pageobjects/MainPage.js

```diff
@@ -0,0 +1,36 @@
+import BasePage from './BasePage.js';
+import ElementAction from '../helpers/ElementAction.js';
+import GetData from '../helpers/GetData.js';
+import Waiters from '../helpers/Waiters.js';
+
+class MainPage extends BasePage {
+  get ['Accept cookie button']() {
+    return '#onetrust-accept-btn-handler';
+  }
+
+  travelStartDateButton = 'button[data-testid="date-display-field-start"]';
+  submitButton = '//button[@type="submit"] | //*[class="sb-searchbox__button"]';
+  calendarButton = '.ed2ff9f661';
+  searchBoxField = '//input[@name="ss"]';
+  getFirstLocation = (locationName) => `//*[normalize-space()="${locationName}"]`;
+  getDataInCalendar = (data) => `//*[@aria-label="${data}"]`;
+
+  async fillTheField(fieldName, text) {
+    await Waiters.waitElementIsDisplayed(element);
+    await ElementAction.setValueInField(element, text);
+    await ElementAction.click(this.getFirstLocation(text));
+    await ElementAction.click(element);
+    await ElementAction.sendKeys(BasePage.enterKey);
+  }
+
+  async selectTheDataFromCalendar(dataOne, dataTwo) {
```

Comment:
Please remove `The` from the method name, and rename parameters' names: `startDate, endDate`.

Reply:
I delete this code, because YAGNI